### PR TITLE
Updated references to datamodels in stpipe,  comments.

### PR DESCRIPTION
**Pull request by jaytmiller**
I updated two modules in stpipe from references to jwst_lib.models to jwst.datamodels.   The changes involved comments and a log message.

Based on feedback from Howard,  I also changed all the existing datamodels imports from absolute to relative.

### CODE REVIEW

File: jwst/stpipe/step.py [hunk 1]
```diff
@@ -602,7 +602,7 @@ def get_reference_file_model(self, input_file, reference_file_type):
 
         Returns
         -------
-        reference_file_model : jwst_lib.models.ModelBase instance
+        reference_file_model : jwst.datamodels.ModelBase instance
             A model to access the contents of the reference file.
         """
         from jwst import datamodels
```
Comment (hbushouse) [inline jwst/stpipe/step.py]: Should this be "from .. import datamodels"?

Comment (jaytmiller) [inline jwst/stpipe/step.py]: I'm pretty sure that import was already there and is not really part of this pull request.  I have no strong beliefs about which is best,  relative or absolute.  Would you like another pull request?

Comment (nden) [inline jwst/stpipe/step.py]: You can change the import in this existing branch and push again to origin. It will show up as a separate commit. But yeah, we want to use relative imports.

Comment (hbushouse) [inline jwst/stpipe/step.py]: The consensus during today's hackathon seemed to be that relative imports were preferred, just in case someone accidentally has something else defined as "jwst".

**Comment by jaytmiller** [comment]
OK,  not sure if you guys saw my additional commit,  but I went in and changed all the existing datamodels imports from absolute to relative.

**Comment by hbushouse** [comment]
👍
